provisioning/jmapAuthentication.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# This script feeds data into the James server for the scenario JmapAuthentication
+#
+# Argment $1 : The cotainer ID of your James server
+
+docker exec $1 java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999 adddomain domain-jmapauthentication.tld
```

Comment:
That's ok for this PR but keep in mind we should connect to James with jmx to do that